### PR TITLE
Added tests for cases incorrect or absent response from HMI

### PR DESCRIPTION
**Pull request by aderiabin**

### CODE REVIEW

File: test_scripts/RC/ButtonPress/012_generic_error_if_invalid_response_from_hmi.lua [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+-- In case:
+-- 1) RC app sends ButtonPress request with valid parameters
+-- 2) and HMI response is invalid
+-- SDL must:
+-- 1) Respond to App with success:false, "GENERIC_ERROR"
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonRC = require('test_scripts/RC/commonRC')
+local commonTestCases = require('user_modules/shared_testcases/commonTestCases')
```
Comment (dboltovskyi) [inline test_scripts/RC/ButtonPress/012_generic_error_if_invalid_response_from_hmi.lua]: @aderiabin commonTestCases is not used

File: test_scripts/RC/ButtonPress/012_generic_error_if_invalid_response_from_hmi.lua [hunk 2]
```diff
@@ -0,0 +1,54 @@
+---------------------------------------------------------------------------------------------------
```
Comment (dboltovskyi) [inline test_scripts/RC/ButtonPress/012_generic_error_if_invalid_response_from_hmi.lua]: @aderiabin Please add these new scripts to appropriate test set

Comment (aderiabin) [inline test_scripts/RC/ButtonPress/012_generic_error_if_invalid_response_from_hmi.lua]: @dboltovskyi Done in d9b11429e11a1c74c14299f624223d0d4c997c15

File: test_scripts/RC/ButtonPress/012_generic_error_if_invalid_response_from_hmi.lua (hunk 1 above)
Comment (aderiabin) [inline test_scripts/RC/ButtonPress/012_generic_error_if_invalid_response_from_hmi.lua]: @dboltovskyi Removed in c284d0d10b10f2e930162fda3a0ff159d27d1cd2

**Pull request merged**